tests/test_serial_comm_utils.py

```diff
@@ -158,7 +158,7 @@ def test_parse_metadata_bytes__returns_metadata_as_dictionary(
 @freeze_time("2021-04-07 13:14:07.234987")
 def test_get_serial_comm_timestamp__returns_microseconds_since_2021_01_01():
     expected_usecs = (
-        datetime.datetime.utcnow() - SERIAL_COMM_TIMESTAMP_EPOCH
+        datetime.datetime.now(tz=datetime.timezone.utc) - SERIAL_COMM_TIMESTAMP_EPOCH
```

Comment:
did this not pass with `datetime.datetime.utcnow()`?

Reply:
`utcnow` is a "naive" datetime object and cannot be compared to an "aware" datetime object https://docs.python.org/3/library/datetime.html#datetime.datetime.now

Reply:
I believe this means aware of timezone offset